web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -155,14 +156,25 @@ function BaseCard({
                 <Badge type={pillType} icon={iconPill} pillText={pillText}></Badge>
               )}
               <div className="text-lg">
-                {isCollapsed ? <HiChevronDown /> : <HiChevronUp />}
+                {isCollapsed ? (
+                  <div data-test-id="collapse-down">
+                    <HiChevronDown />
+                  </div>
+                ) : (
+                  <div data-test-id="collapse-up">
+                    <HiChevronUp />
+                  </div>
+                )}
```

Comment:
I couldn't add the `data-test-id` to the `HiChevronUp` and `HiChevronDown`, so this was a workaround to be able to test which icon was showing. Feel free to share it if you have ideas on how to do it differently:)

Reply:
This is probably the easiest method, the other way probably be to try and extract some ID out of the SVG icon to use that but it sounds messy to just explain it in this comment...